docs/source/config_file.rst

```diff
@@ -323,8 +323,8 @@ Platform configuration
     :type: string
 
     Specifies the Python version used to parse and check the target
-    program.  The string should be in the format ``DIGIT.DIGIT`` --
-    for example ``2.7``.  The default is the version of the Python
+    program.  The string should be in the format ``N.NN`` --
```

Comment:
If we want to more precise, what about making it "... in the format ``N.N`` or ``N.NN`` -- ..." instead?